spec/minimal_repo.rb

```diff
@@ -5,13 +5,7 @@ module MinimalRepo
   class << self
     DIRECTORIES = [
       '.git',
-      'config',
-      'hosts',
-      'genders',
-      'dhcp',
       'files',
```

Comment:
I've made this change, let me know what you think: https://github.com/alces-software/metalware/commit/86b15e8acc04ca284e520d5db5f6b7091b135dec.